### ISSUE TITLE
Group "/" and "" routes are separate and should be the same

### ISSUE THREAD

**Issue by briangwaltney** [elysiajs/elysia#71]
```ts
.group("users", app =>
    app.get("/", ()=>{
       return true
    }
)
```

The above code will generate a 404 error when going to `/users`. The trailing slash must be added to find the route. `/users/` is required for the routing to be found.

This is also the situation regardless of the group.

If the path is
```ts
.get("/users", ()=>{})
```
going to `/users/` also fails.

I think `/users` should default to the `"/users/"` route and vice versa.

**Comment by SaltyAom** [comment]
This will be configurable on an upcoming Elysia 0.6.0 via config.strictPath.

Please leave this issue open until Elysia 0.6.0 stable is released.

**Comment by SaltyAom** [comment]
This should be fixed in Elysia 0.6, see [Strict Path](https://elysiajs.com/blog/elysia-06.html#strict-path)
